logistics/scripts/modules/database/database-management.psm1

```diff
@@ -148,16 +148,15 @@ function Get-DbConnectionStringBuilderFromConfig {
         [String[]] $connectionStringName
     )
 
-    [System.Xml.XmlDocument]$configXml = Get-Content $configFile
-
     $csbs = @{ }
 
-    foreach ($connStrEntry in $configXml.configuration.connectionStrings.add) {
-        $csb = New-Object System.Data.Common.DbConnectionStringBuilder
-        # using set_ConnectionString correctly uses the underlying C# setter functionality resulting in a dictionary of connection string properties
```

Comment:
I'd like to keep this comment since it is possible to set the connection string value using `$csb.connectionString = ...` but this won't create the dictionary needed by much or our code.

Reply:
Rolled back the comment